tools/minikube/templates/app-deployment.yaml

```diff
@@ -92,6 +92,17 @@ spec:
                 configMapKeyRef:
                   name: ansible-controller-env
                   key: PINAKES_CONTROLLER_VERIFY_SSL
+            - name: PINAKES_INSIGHTS_TRACKING_STATE
+              value: "True"
+            - name: PINAKES_INSIGHTS_URL
+              valueFrom:
+                configMapKeyRef:
+                  name: ansible-controller-env
+                  key: PINAKES_CONTROLLER_URL
```

Comment:
@hsong-rh should this be PINAKES_INSIGHTS_URL?

Reply:
Yes, changed to read the info from its own configMap `ansible-insights-env`